src/app/components/terms/terms.component.ts

```diff
@@ -0,0 +1,20 @@
+import { Component, OnInit } from '@angular/core';
+import { TermsService } from 'src/app/services/terms/terms.service';
+import { Observable } from 'rxjs';
+@Component({
+  selector: 'app-terms',
+  templateUrl: './terms.component.html',
+  styleUrls: ['./terms.component.scss']
+})
+export class TermsComponent implements OnInit {
+
+  terms: Observable<any>;
+  constructor(private termsService: TermsService) {
+  }
+
+  ngOnInit() {
```

Comment:
@CryceTruly , it's good practice to unsubscribe from any service especially if the component is not going be mounted in another view. Take advantage of the `ngDestroy()` lifecycle hook for this

Reply:
Thanks, @malep2007, I have restructured the component class to the pages folder.